internal/appconfig/patches.go

```diff
@@ -114,9 +114,33 @@ func _patchEnv(raw any) (map[string]string, error) {
 func patchProcesses(cfg map[string]any) (map[string]any, error) {
 	if raw, ok := cfg["processes"]; ok {
 		switch cast := raw.(type) {
-		case []any, []map[string]any:
-			// GQL GetConfig returns an empty array when there are not processes
-			delete(cfg, "processes")
+		case []any:
+			processes := []map[string]any{}
+			for _, item := range cast {
+				if v, ok := item.(map[string]any); ok {
+					processes = append(processes, v)
+				}
+			}
+			cfg["processes"] = processes
+			return patchProcesses(cfg)
+
+		case []map[string]any:
```

Comment:
Should `[]any` go below `[]map[string]any`? I feel like this would always stop at the first option here, but I could be wrong.

Reply:
I was expecting that too but it doesn't, the type match is exact and the test case covers it.